src/app/shared/components/support-information/support-information.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Http } from '@angular/http';
+import { Converter } from 'showdown';
+import { UserTagService } from '../../services/tags/user-tag.service';
+import { Observable } from 'rxjs/Observable';
+
+@Component({
```

Comment:
Maybe a separate module should be created for this component and the functionality described in this ticket